tailwind.config.ts

```diff
@@ -23,49 +23,49 @@ const config: Config = {
     extend: {
       colors: {
         light: {
-          primary: '#1a202c',
-          secondary: '#f2b034',
-          tertiary: '#43a677',
-          accent: '#f19a2e',
-          background: '#FEF5EA',
-          text: '#184123',
-          muted: '',
-          link: '#91d9bf',
-          border: '#3d5b50',
-          placeholder: '',
-          hover: '',
-          active: '',
-          error: '',
-          success: '',
-          warning: '#d9580d',
-          info: '',
+          primary: '#2B3C47',
+          secondary: '#F2B034',
+          tertiary: '#43A677',
+          accent: '#F19A2E',
+          background: '#F9FCFC',
+          text: '#2B3C47',
+          muted: '#A8AFB4',
+          link: '#91D9BF',
+          border: '#3D5B50',
+          placeholder: '#D9D9D9',
+          hover: '#307654',
+          active: '#3D976C',
+          error: '#E4190C',
+          success: '#43A677',
+          warning: '#D9580D',
+          info: '#D9D9D9',
           transparent: 'transparent',
           current: 'currentColor',
-          black: '#2c1d15',
-          white: '#fdfefe',
+          black: '#2C1D15',
+          white: '#FDFEFE',
         },
         dark: {
-          primary: '#013b3f',
+          primary: '#013B3F',
           secondary: '#096766',
-          tertiary: '#012c2f',
-          accent: '#ffa51b',
+          tertiary: '#012C2F',
+          accent: '#FFA51B',
           background: '#013B3F',
           text: '#EAF0EB',
-          muted: '#e6ebec',
-          link: '',
+          muted: '#E6EBEC',
+          link: '#D9D9D9',
           border: '#938F99',
-          placeholder: '',
+          placeholder: '#D9D9D9',
           icon: '',
```

Comment:
This property isn't on the color guide and I'm not sure why it's only in the dark theme. We could potentially just remove for now.

Reply:
Noted, I will merge for now add address this in the styling PR, thank you @ejwoods